components/permit-holders/GuardianInformationCard.tsx

```diff
@@ -6,26 +6,25 @@ type GuardianInformationProps = {
   guardian: Guardian;
 };
 
-export default function GuardianInformationCard(props: GuardianInformationProps) {
-  const { guardian } = props;
+export default function GuardianInformationCard({ guardian }: GuardianInformationProps) {
   return (
     <PermitHolderInfoCard colSpan={7} header={`Guardian's Information`}>
       <Divider pt="24px" />
 
       <VStack spacing="12px" align="left" paddingTop="24px">
         <Box>
           <Text as="p" textStyle="body-regular">
-            {`${guardian.firstName} ${guardian.lastName}`}
+            {`${guardian?.firstName || ''} ${guardian?.lastName || ''}`}
```

Comment:
Just a note to remove the optional chaining now that `guardian` is a mandatory prop